Launching-new-features.md

```diff
@@ -3,7 +3,8 @@ Please use the following process for launching major new features that lead to a
 - Before doing anything else, please:
     - Make sure e2e tests are present for your feature.
     - Make sure it is easy to toggle on and off, in case it needs to be aborted for the release. This implies proper gating of views, controllers, etc.
```

Comment:
Should we be more prescriptive here about using dynamic feature gating? /cc @nithinrdy

Reply:
Hey, sorry for the delay.

a. Yes, perhaps we could add a link to the feature gating wiki page, and
b. We could add a note right after which talks about writing E2E tests for gated features, something like this:

`To write E2E tests for features gated behind a feature flag, you may need to use the 'enableFeatureForProd'/'enableFeatureForDev' utility functions for the admin page to first enable the required flag, and then proceed to perform the testing.`

Reply:
Done.